index.html

```diff
@@ -1,166 +0,0 @@
-<!DOCTYPE html>
-<html lang="fr">
-    <head>
-        <meta charset="utf-8"/>
-        <meta name="description" content="CV en ligne de David Cailleau">
```

Comment:
Tu peux écrire un texte plus pertinent, cette balise est importante pour le SEO. N'hésite pas à chercher des articles sur le web pour avoir des conseils, par exemple https://www.yakaferci.com/balise-meta-description/

Reply:
Tu peux aussi avoir des conseils de base avec cette extension https://chrome.google.com/webstore/detail/seo-meta-in-1-click/bjogjfinolnhfhkbipphpdlldadpnmhc?hl=fr